frontend/src/components/dialogs/GDialog.vue

```diff
@@ -202,6 +203,23 @@ export default {
         resolve(value)
       }
       this.visible = false
+    },
+    showScrollBar () {
+      const contentCardRef = this.$refs.contentCard
+      if (!contentCardRef || !contentCardRef.clientHeight) {
+        this.$nextTick(() => this.showScrollBar())
```

Comment:
Potential endless loop?

Reply:
hmm yes could be if the ref is never set. However, this should not be the case. Anyway, if you find a better solution it would be highly appreciated

Reply:
see comment below

Reply:
added circuit breaker